gusto/core/function_spaces.py

```diff
@@ -109,9 +114,29 @@ def add_space(self, name, space, overwrite_space=False):
 
         setattr(self, name, space)
 
-    def create_space(self, name, family, degree, overwrite_space=False):
+        # set the continuity of the space - for extruded meshes specify both directions
+        if continuity is None:
+            from gusto.time_discretisation.wrappers import is_cg
```

Comment:
do we want to put the `is_cg` function somewhere else now that it is not only used in the time discretisations?